src/clojurians_log/db/queries.clj

```diff
@@ -109,6 +109,15 @@
        db
        ids))
 
+(defn user-profile
+  "returing all the data about the user acording to the user id slack number"
+  [db id]
+  (first (d/q '[:find (pull ?user [*])
+                :in $ ?id
+                :where [?user :user/slack-id ?id]]
+              db
+              id)))
```

Comment:
Is this returning a single map or a sequence of maps? If it's a sequence can we change it to just return a single map? that will make lookup later more straightforward. I think adding a `.` at the end of line 115 might be enough.